compiler-rt/lib/mpk_untrusted/tests/unit/alloc_site_handler_tests.cpp

```diff
@@ -69,9 +72,81 @@ TEST(getAllocSite, ValidBetweenAddress) {
 }
 
 TEST(getAllocSite, PointerArithmeticOverflowWraps) {
-  auto ptr = (rust_ptr)(-1); // 0xffffffffffffffff
+  auto ptr = (rust_ptr)(-1); // max pointer size
   auto newAllocSite = std::make_shared<AllocSite>(ptr, sizeof(uint64_t), 1);
   EXPECT_FALSE(newAllocSite->containsPtr(++ptr));
 }
 
+TEST(faultingAllocs, addFaultAlloc) {
+  auto handle = AllocSiteHandler::getOrInit();
+  auto ptr = (rust_ptr)malloc(sizeof(uint64_t));
+  auto newAllocSite = std::make_shared<AllocSite>(ptr, sizeof(uint64_t), 1);
```

Comment:
I know this is an existing test, but what is the 1 for?

Reply:
Switched the 1 for a constant named `PKEY_DEFAULT_VALUE` to indicate that it is just a default for testing.